### PR TITLE
ci: fix delete.bats for GHA

### PR DESCRIPTION
**Pull request by kolyshkin**
_TL;DR: this is a forward-port of https://github.com/opencontainers/runc/pull/3538/commits/f46c0dad655e33cf29d38da94b265d982a835f88 (part of #3538) to main branch, fixing a CI flake caused by a GHA CI env peculiarity._

A couple of test cases in delete.bats check that a particular cgroup
exists (or doesn't exist) using find. This is now resulting in errors
like these:

        find: ‘/sys/fs/cgroup/blkio/azsec’: Permission denied
        find: ‘/sys/fs/cgroup/blkio/azsec_clamav’: Permission denied
        find: ‘/sys/fs/cgroup/cpu,cpuacct/azsec’: Permission denied
        find: ‘/sys/fs/cgroup/cpu,cpuacct/azsec_clamav’: Permission denied
        find: ‘/sys/fs/cgroup/memory/azsec’: Permission denied
        find: ‘/sys/fs/cgroup/memory/azsec_clamav’: Permission denied

leading to test case failures.

Apparently, GHA runs something else on a test box, so we get this.

To fix, ignore non-zero exit code from find, and redirect its stderr
to /dev/null.

### CODE REVIEW

**Review by AkihiroSuda — DISMISSED** [review]

**Comment by kolyshkin** [comment]
@thaJeztah PTAL

**Review by thaJeztah — DISMISSED** [review]
LGTM

**Comment by thaJeztah** [comment]
Should we disable this check? Not sure if it's really tenable to require all PR's to be up-to-date;
Screenshot 2022-11-01 at 16 29 17

**Comment by kolyshkin** [comment]
> This actually helps to ensure that two PRs that both took a long time to merge won't break one another. I've seen a few cases of that in the past, and now we have a mechanism (update with rebase button) which rebases the PR without losing LGTMs.
> 
> So, I am going to rebase it and, if CI is all nice and green, merge it.

Overall, this is an inherent issue with GitHub. There's no way to see what happened in between force pushes, so we can't have "leave LGTMs after force-push" enabled.

For this and many other reasons, I'd switch to gerrit, but I'm not sure other @opencontainers/runc-maintainers concur.

**Comment by kolyshkin** [comment]
> and now we have a mechanism (update with rebase button) which rebases the PR without losing LGTMs.

Hmm, either something has changed or I did a mistake a few minutes ago, but this is no longer working the way it's supposed to (meaning, I can rebase the branch without losing LGTMs). Oh my :(

**Comment by kolyshkin** [comment]
OTOH it just worked for https://github.com/opencontainers/runc/pull/3635, so maybe if there are some noticeable changes during the rebase, LGTMs are lost, and if not, they are retained.

**Review by mrunalp — DISMISSED** [review]

**Review by AkihiroSuda — APPROVED** [review]

**Review by mrunalp — APPROVED** [review]

**Comment by kolyshkin** [comment]
@thaJeztah @AkihiroSuda PTAL

**Review by AkihiroSuda — APPROVED** [review]

**Review by hqhq — APPROVED** [review]
